iree/base/internal/flatcc.h

```diff
@@ -30,20 +30,20 @@
 // clang-format off
 
 #include "flatcc/reflection/flatbuffers_common_reader.h"
-#include "iree/base/flatcc_reader.h"
```

Comment:
update header guard please

from
```c++
#ifndef IREE_BASE_FLATCC_H_
#define IREE_BASE_FLATCC_H_
```

to
```c++
#ifndef IREE_BASE_INTERNAL_FLATCC_H_
#define IREE_BASE_INTERNAL_FLATCC_H_
```

Reply:
good catches!